qa/integration-tests/src/test/java/io/zeebe/broker/it/network/LargeMessageSizeTest.java

```diff
@@ -24,7 +24,7 @@
 
 public final class LargeMessageSizeTest {
 
-  private static final DataSize MAX_MESSAGE_SIZE = DataSize.ofMegabytes(4);
+  private static final DataSize MAX_MESSAGE_SIZE = DataSize.ofMegabytes(5);
```

Comment:
Why do you need this change?

Reply:
The encoded size is larger now so the size of the entry exceeded 5Mb